src/ray/core_worker/lib/java/jni_utils.h

```diff
@@ -41,15 +41,13 @@ extern jfieldID java_native_ray_object_data;
 extern jfieldID java_native_ray_object_metadata;
 
 /// Throws a Java RayException if the status is not OK.
-/// \return True if throwed exception, otherwise false.
-inline bool ThrowRayExceptionIfNotOK(JNIEnv *env, const ray::Status &status) {
-  if (!status.ok()) {
-    env->ThrowNew(java_ray_exception_class, status.message().c_str());
-    return true;
-  } else {
-    return false;
+#define ThrowRayExceptionIfNotOK(env, status, ret)                           \
```

Comment:
nit, macros should use upper case letters.  maybe `THROW_EXCEPTION_AND_RETURN_IF_NOT_OK`?

Reply:
My bad. Updated.